### ISSUE TITLE
Randomite Only Drops Cobble

### ISSUE THREAD

**Issue by mysteryblock** [kegare/Cavern#50]
As the title implies, after mining 100-150+ blocks of randomite in my world It only ever drops cobble. As such I cannot obtain any thought orbs. 

Minecraft 1.12.2
Forge: 14.23.0.2512
Optifine: HD_U_C6
Cavern: 1.10.2

**Comment by C0rn3j** [comment]
Doesn't happen on my server, I use the same version of mod/MC/forge. You should post your full mod list. 

Here is mine, you should pay extra attention to any extra mods you may have over me.

https://paste.c0rn3j.com/?69f7b5e82d139d58#Pb25KD4XebwkbirUr16XGATyXq5ncbQY5cHqOrYCucU=

**Comment by mysteryblock** [comment]
https://drive.google.com/open?id=0BxszNcfHzQJNQ2FUQlp2MzNZUW8

Link to my modlist at the time of the posting ^

**Comment by mysteryblock** [comment]
I have better with mods / better with addons installed, and that seems to conflict with many things. Not sure if it's the culprit but I noticed you do not have it in your mod list.

**Comment by mysteryblock** [comment]
I don't know, but I feel it might be this mod: https://minecraft.curseforge.com/projects/cavern-tweaks?gameCategorySlug=mc-mods&projectID=268582

Going to remove it, test, and let you know.

**Comment by mysteryblock** [comment]
Yep, removing that mod fixed the issue. Randomite now works. I'll let that mod author know.

**Comment by MCJediYoda** [comment]
If caverntweaks is dropping cobble, then you don't have the items configured in caverntweaks.cfg.  Here is an example of the formatting for how the items should be added in caverntweaks.cfg.  This is assuming you are using the latest version also.

# <modid>:<item>|<meta>|<quantity>|<weight> [default: []]

S:"Randomite Items" <
        cavern:cave_item|4|1|1
        cavern:cave_item|0|1|10
        cavern:cave_item|2|1|3
        cavern:cave_item|5|1|3
        minecraft:coal|0|1|10
        minecraft:stick|0|1|25
        minecraft:flint|0|1|10
        minecraft:bone|0|1|20
        minecraft:redstone|0|1|5
        minecraft:rotten_flesh|0|1|20
        minecraft:name_tag|0|1|5
        minecraft:gold_nugget|0|3|5
>

Let me know if that helps.
